docker-compose/docker-compose-dev-mode.yml

```diff
@@ -1,26 +1,24 @@
 version: '2'
 
 services:
-  coordinator:
-    image: stargateio/coordinator-dse-next:${SGTAG}
+  dse:
```

Comment:
The concept of a "dev mode" docker compose doesn't make sense any longer. We should just remove the whole file.

Reply:
Same as above, we don't start data-api in this script.